streaming/README.md

```diff
@@ -227,6 +233,9 @@ The receiving end (proxy or parent) logs entries like these:
 
 For Netdata v1.9+, streaming can also be monitored via `access.log`.
 
+There are no any configuration options for enabling or disabling stream data compression.
```

Comment:
I believe this sentence will create confusion for users, because we have `enable compression` option.

Reply:
Thanks for spotting it. I removed this sentence.